build.sh

```diff
@@ -172,28 +253,51 @@ fi
 
 ################################################################################
 # pylibwholegraph
-if hasArg pylibwholegraph; then
+if buildAll || hasArg pylibwholegraph; then
+    if hasArg --clean; then
+        cleanPythonDir ${REPODIR}/python/pylibwholegraph
+    fi
+
     # setup.py and cmake reference an env var LIBWHOLEGRAPH_DIR to find the
     # libwholegraph package (cmake).
     # If not set by the user, set it to LIBWHOLEGRAPH_BUILD_DIR
     LIBWHOLEGRAPH_DIR=${LIBWHOLEGRAPH_DIR:=${LIBWHOLEGRAPH_BUILD_DIR}}
     if ! hasArg --compile-cmd; then
-        cd ${REPODIR}/pylibwholegraph
+        cd ${REPODIR}/python/pylibwholegraph
+        echo "Changed to: ${PWD} and ${PYTHON}"
         env LIBWHOLEGRAPH_DIR=${LIBWHOLEGRAPH_DIR} \
-        ${PYTHON} ${REPODIR}/pylibwholegraph/setup.py build_ext --inplace \
+        ${PYTHON} setup.py build_ext --inplace \
             --build-type=${BUILD_TYPE} \
             ${EXTRA_CMAKE_ARGS}
         if ! hasArg -n; then
             env LIBWHOLEGRAPH_DIR=${LIBWHOLEGRAPH_DIR} \
-            ${PYTHON} ${REPODIR}/pylibwholegraph/setup.py install \
+            ${PYTHON} setup.py install \
                 --build-type=${BUILD_TYPE} \
                 ${EXTRA_CMAKE_ARGS}
         fi
     else
         # just invoke cmake without going through scikit-build
         env LIBWHOLEGRAPH_DIR=${LIBWHOLEGRAPH_DIR} \
-        cmake -S ${REPODIR}/pylibwholegraph -B ${REPODIR}/pylibwholegraph/_skbuild/build \
+        cmake -S ${REPODIR}/python/pylibwholegraph -B ${REPODIR}/python/pylibwholegraph/_skbuild/build \
            -DCMAKE_BUILD_TYPE=${BUILD_TYPE} \
             ${EXTRA_CMAKE_ARGS}
     fi
 fi
+
+################################################################################
+# Build the docs
+if hasArg docs; then
+    if [ ! -d ${LIBCUGRAPH_BUILD_DIR} ]; then
+        mkdir -p ${LIBCUGRAPH_BUILD_DIR}
+        cd ${LIBCUGRAPH_BUILD_DIR}
+        cmake -B "${LIBCUGRAPH_BUILD_DIR}" -S "${REPODIR}/cpp" \
+              -DCMAKE_INSTALL_PREFIX=${INSTALL_PREFIX} \
+              -DCMAKE_BUILD_TYPE=${BUILD_TYPE} \
+              ${CMAKE_GENERATOR_OPTION} \
+              ${CMAKE_VERBOSE_OPTION}
+    fi
+    cd ${LIBCUGRAPH_BUILD_DIR}
+    cmake --build "${LIBCUGRAPH_BUILD_DIR}" -j${PARALLEL_LEVEL} --target docs_cugraph ${VERBOSE_FLAG}
+    cd ${REPODIR}/docs/cugraph
```

Comment:
Seems here should be `LIBWHOLEGRAPH_BUILD_DIR` instead of `LIBCUGRAPH_BUILD_DIR`, and also maybe also wholegraph in docs?
```suggestion
    if [ ! -d ${LIBWHOLEGRAPH_BUILD_DIR} ]; then
        mkdir -p ${LIBWHOLEGRAPH_BUILD_DIR}
        cd ${LIBWHOLEGRAPH_BUILD_DIR}
        cmake -B "${LIBWHOLEGRAPH_BUILD_DIR}" -S "${REPODIR}/cpp" \
              -DCMAKE_INSTALL_PREFIX=${INSTALL_PREFIX} \
              -DCMAKE_BUILD_TYPE=${BUILD_TYPE} \
              ${CMAKE_GENERATOR_OPTION} \
              ${CMAKE_VERBOSE_OPTION}
    fi
    cd ${LIBWHOLEGRAPH_BUILD_DIR}
    cmake --build "${LIBWHOLEGRAPH_BUILD_DIR}" -j${PARALLEL_LEVEL} --target docs_wholegraph ${VERBOSE_FLAG}
    cd ${REPODIR}/docs/wholegraph
```